README.md

```diff
@@ -55,7 +55,7 @@ Supported Compilers
 
 This library uses C++11 (or newer when available).
 
-|Accelerator Back-end|gcc 4.9.2|gcc 5.2|clang 3.5/3.6|clang 3.7|MSVC 2015|
+|Accelerator Back-end|gcc 4.9.2|gcc 5.3|clang 3.5/3.6|clang 3.7|MSVC 2015|
```

Comment:
I have updated this, as the Travis CI Ubuntu now provides gcc 5.3. Therefore, gcc 5.2 is not tested anymore. This is no problem as it is only a bugfix gcc release.